workflow-pacta-cop.Rmd

```diff
@@ -223,8 +223,7 @@ set_project_parameters(file.path(par_file_path, "ProjectParameters.yml"))
 analysis_inputs_path <- set_analysis_inputs_path(twodii_internal, data_location_ext, dataprep_timestamp)
 
 inc_meta_portfolio <- FALSE
-write_csv(portfolio_peers, file = file.path(raw_input_path, paste0(project_name, "_Input.csv")))
-
+write_csv(portfolio_peers, file.path(raw_input_path, paste0(project_name, "_Input.csv")))
```

Comment:
Here I prefer to match by position to avoid exposing the internal packages PACTA.analysis.